.github/workflows/publish-chrome-development.yml

```diff
@@ -0,0 +1,106 @@
+name: publish-chrome-development
+on:
+  release:
+    types: [prereleased]
+  workflow_dispatch:
+    inputs:
+      attemptNumber:
+        description: "Attempt number"
+        required: false
+        default: "1"
+      maxAttempts:
+        description: "Max attempts"
+        required: false
+        default: "10"
+      environment:
+        description: "publish-on-webstore job environment"
+        required: false
+        default: ""
+jobs:
+  upload-on-webstore:
+    runs-on: ubuntu-latest
+    environment: ${{ github.event.inputs.environment }}
+    outputs:
+      result: ${{ steps.webStorePublish.outcome }}
+      releaseUploadUrl: ${{ steps.getZipAsset.outputs.releaseUploadUrl }}
+    steps:
+      - name: Get the next attempt number
+        id: getNextAttemptNumber
+        uses: cardinalby/js-eval-action@v1
+        env:
+          attemptNumber: ${{ github.event.inputs.attemptNumber }}
+          maxAttempts: ${{ github.event.inputs.maxAttempts }}
+        with:
+          expression: |
+            {
+              const 
+                attempt = parseInt(env.attemptNumber),
+                max = parseInt(env.maxAttempts);
```

Comment:
I wonder if you even need to use `parseInt` here if `attemptNumber` and `maxAttempts` were set to be integer already in the `workflow_dispatch` inputs

Reply:
Not sure, but I stole this from another workflow so there's probably a reason it's like this. If I figure out how to test this I can try removing them.

Reply:
Turns out that it is necessary. All the env inputs are strings from the perspective of `js-eval-action`.